build-package/certifaiReferenceModelServer/insurance_auto_insurance_claims/supportVectorRegression/train.py

```diff
@@ -2,12 +2,12 @@
 import json
 import sys
 import random
-from sklearn.ensemble import RandomForestRegressor
+from sklearn.svm import SVR
 from sklearn.metrics import r2_score
 import pandas as pd
 import numpy as np
-from certifaiReferenceModelServer.insurance_auto_insurance_claims.common_utils.train_utils import CategoricalEncoder
-from certifaiReferenceModelServer.utils.encode_decode import pickle_model
+from insurance_auto_insurance_claims.common_utils.train_utils import CategoricalEncoder
+from utils.encode_decode import pickle_model
```

Comment:
Small issue, but I think you need to keep the `certifaiReferenceModelServer.` prefix for these imports. I think if you exclude it the import statement will fail unless these files are relative to your local path

Another option might be writing it as: `from .insurance_auto_insurance_claims.common_utils.train_utils import CategoricalEncoder` - I think the dot specifies that the import relative to where this file lives, which would normally be in `site-packages/`  along with the file..

(Python import statements are surprisingly tricky...)

Reply:
Thanks! That was an unintentional change.

Reply:
Fixed (tested by installing package and running server from installed package)